kodilla-testing/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java'
+}
+
+group = 'com.kodilla'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.8.2'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.8.2'
+    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '4.4.0'
+
+}
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
to jest czesc innego zadania. |

Po kazdym merge:
- zrob update lokalnego mastera
- wychodz z nowym branchem zawsze z mastera